### PR TITLE
fix: question mark button for environment quick help

### PR DESCRIPTION
**Pull request by itsmdasifraza**
## Description
fix: question mark button for environment quick help
Fixes issue #157
Fixes issue #158 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

### CODE REVIEW

File: src/lib/components/environments/enviroments-panel/sub-components/quick-help/QuickHelp.svelte [hunk 1]
```diff
@@ -64,8 +64,9 @@
   </p>
   <p><img class="w-100" src={environment} alt="" /></p>
   <p class="heading-para text-textColor">
-    All the global variables are marked with G. Where as all local variables
-    within an environment are marked with E.
+    All the global variables are marked with <span class="global-env-G">G</span>
```
Comment (parthdalaltech) [inline src/lib/components/environments/enviroments-panel/sub-components/quick-help/QuickHelp.svelte:67]: move it to const in constants file and import it here

**Review by LordNayan — DISMISSED** [review]

**Review by parthdalaltech — CHANGES_REQUESTED** [review]

File: src/lib/components/environments/enviroments-panel/sub-components/quick-help/QuickHelp.svelte (hunk 1 above)
Comment (itsmdasifraza) [reply src/lib/components/environments/enviroments-panel/sub-components/quick-help/QuickHelp.svelte]: Resolved

**Review by parthdalaltech — APPROVED** [review]

**Pull request merged**